velox/docs/monitoring/stats.rst

```diff
@@ -39,3 +39,7 @@ Memory Arbitration
      - kByte
      - The reclaimed memory bytes of an operator during the memory arbitration.
        This stats only applies for spillable operators.
+   * - joinTableBuildWallNanos
```

Comment:
This metric doesn't belong to "Memory Arbitration".

Reply:
Add a `Task Execution` section.